waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -761,6 +761,7 @@
                     :authorizer {:kind :default
                                  :default {:factory-fn 'waiter.authorization/noop-authorizer}}
                     :cluster-name "waiter"
+                    :custom-options '{a 1 b "two"}
```

Comment:
```suggestion
                    :custom-options {:a 1 :b "two"}
```

Why is there a leading tick in front of the map? Also, should we require the keys to be keywords?

Reply:
I don't see the value in restricting the keys to be keywords. I don't think we enforce that anywhere else, and I don't like restricting the user when I don't have a strong reason for doing so. However, I did change the test map to use keywords, as you suggested, for consistency.